src/openvic-simulation/diplomacy/CountryRelation.cpp

```diff
@@ -1,54 +1,414 @@
-#include "CountryRelation.hpp"
+#include "openvic-simulation//diplomacy/CountryRelation.hpp"
```

Comment:
```suggestion
#include "CountryRelation.hpp"
```
The cpp of a type imports it's hpp the short way in the rest of the repo.

Reply:
Why would this file deviate from the rest?
Also why // ?